gulpfile.js

```diff
@@ -39,7 +39,7 @@ function gulp_test() {
     return gulp.src(tests)
     .pipe(mocha({
         log: true,
-        timeout: 60000,
+        timeout: 1200000,
```

Comment:
Oh you're right. I think you're off by 10 then though, you have 1200000 ms / 1200 sec == 20 mins.